apps/pie-storybook/stories/pie-checkbox-group.stories.ts

```diff
@@ -29,6 +29,13 @@ const checkboxGroupStoryMeta: CheckboxGroupStoryMeta = {
             description: 'The visible label for the checkbox group.',
             control: 'text',
         },
+        isInline: {
+            description: 'Inline positioning of checkbox items.',
```

Comment:
```suggestion
            description: 'Inline (horizontal) positioning of checkbox items.',
```

Reply:
Personally I feel like inline already conveys enough meaning here, as opposed to block

Reply:
I agree with Xander

Reply:
I agree for those that have knowledge of inline and block - some of our consumers will be less frontend savvy, and as this is a description (effectively a comment), I think it's beneficial to add more context.

Happy for the name to still be isInline, just think the extra clarity can be good.